### ISSUE TITLE
one training problem

### ISSUE THREAD

**Issue by SRRon** [princewang1994/RFCN_CoupleNet.pytorch#5]
hello! first thanks for your sharing about this work! I meet one problem when training: 
during the first epoch, the loss convergent normal. but after first epoch, the loss won't continue to convergent even trained 99 epoch, decrease lr can't help. And when i use the model of first epoch to test, i find the feature map after self.RCNN_base(im_data) is all zero!  I think it means the weight of conv is all zero after first epoch training. Does it means the training fall into local minimum or something else wrong? Sincerely thanks for your help!
